integrations/Pendo/browser.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable lines-between-class-members */
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class Pendo {
+  constructor(config, analytics) {
+    this.analytics = analytics;
+    this.apiKey = !config.apiKey ? "" : config.apiKey;
+    this.isDebugMode = config.enableDebugMode;
+    this.name = "PENDO";
+    this.setIntervalHandler = undefined;
+    logger.debug("Config ", config);
+  }
+
+  init() {
+    window.pendo = {};
+    window.pendoCli = undefined;
+    window.pendo.apiKey = this.apiKey;
+    ScriptLoader(
+      "pendo-fs",
+      `https://cdn.pendo.io/agent/static/${this.apiKey}/pendo.js`
+    );
+    logger.debug("===in init Pendo===");
+
+    this.setIntervalHandler = setInterval(this.initPendo.bind(this), 1000);
+  }
+
+  initPendo() {
+    if (this.isReady()) {
+      window.pendoCli = window.pendo;
+      clearInterval(this.setIntervalHandler);
+      if (this.isDebugMode) {
+        this.enableDebugging();
+      } else {
+        this.disableDebugging();
+      }
+    }
+  }
+  isLoaded() {
+    logger.debug("in PENDO isLoaded");
+    return !!window.pendoCli;
+  }
+
+  isReady() {
+    return window.pendo && window.pendo.isReady && window.pendo.isReady();
+  }
+
+  identify(rudderElement) {
+    let visitorObj = {};
+    visitorObj.id =
+      rudderElement.message.userId || rudderElement.message.anonymousId;
+    if (rudderElement.message.context.traits) {
+      visitorObj = {
+        ...visitorObj,
+        ...rudderElement.message.context.traits,
+      };
+    }
+
+    this._identify({ visitor: visitorObj });
+  }
+
+  track(rudderElement) {
+    window.pendoCli.track(rudderElement.message.event);
+  }
+
+  group(rudderElement) {
+    let accountObj = {};
+    accountObj.id =
+      rudderElement.message.groupId || rudderElement.message.anonymousId;
+    if (rudderElement.message.context.traits) {
+      accountObj = {
+        ...accountObj,
+        ...rudderElement.message.context.traits,
+      };
+    }
+    this._identify({ account: accountObj });
+  }
+
+  _identify(obj) {
```

Comment:
As sayan da had commented in the pond doc : 

can you check if this helps: https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/pendo/lib/index.js#L124

Are we doing this? if not what is the difference between these two calls?

Reply:
Yes I am doing same what Sayan da suggested.